ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -307,13 +311,17 @@ void testShowPrincipals() {
 
     private boolean isDatabasePrincipal(Client localQueryClient) {
         KustoOperationResult result = null;
-        boolean found = false;
         try {
-            result = localQueryClient.execute(databaseName, String.format(".show database %s principals", databaseName));
-            // result = localQueryClient.execute(databaseName, String.format(".show version"));
+            result = localQueryClient.executeMgmt(DB_NAME, String.format(".show database %s principals", DB_NAME));
         } catch (Exception ex) {
-            Assertions.fail("Failed to execute show database principals command", ex);
+            Assertions.fail("Failed to execute show database principals command. " +
+                    "Is USERNAME_HINT set to your email in your environment variables?", ex);
```

Comment:
move this as comment to the testCreateWithUserPrompt and testCreateWithConnectionStringAndUserPrompt- This method here is not used only by the prompt test

Reply:
done :)